cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/DebugPhaseTapChanger.java

```diff
@@ -59,7 +64,7 @@ private void explorePhaseTapPositions(
                     tx.getTerminal2().getBusView().getBus().getV(),
                     tx.getTerminal2().getBusView().getBus().getAngle()));
         }
-        int backup = tx.getPhaseTapChanger().getTapPosition();
+        int backup = tx.getPhaseTapChanger().getTapPosition().orElseThrow(() -> new PowsyblException("SCADA network not supported"));
```

Comment:
Just an idea: can you provide the error message `SCADA validation level not supported` in a way or another through the API ?